tests-wtr/dom-actions/dom-actions.element-selector.ts

```diff
@@ -0,0 +1,94 @@
+import { expect } from '@esm-bundle/chai';
+import { utils } from "./utils";
+
+// must be run from dom-actions.element-selector.html
+describe('elementSelector', () => {
```

Comment:
This might feel a bit strange - I'm not actually testing elementSelector, but its two underlying methods - it made more sense to me to keep those in this single file.

Reply:
You could also just call `elementSelector()` in all these cases. Maybe we should treat the underlying methods "private", I don't think there's much sense calling them directly in real life.